packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.8
+
+* Ensures `buffering` state is only removed when the browser reports enough video
+  has been buffered for full playback (`onCanPlayThrough`).
```

Comment:
Nit: s/for full playback/that the video can likely play through without stopping/